README.md

```diff
@@ -3,4 +3,4 @@ my demo repository where I'm following the help directions
 
 Now I'm adding some stuff to this file so that I can play around with the behavior in the diff and make some suggestions and comments. 
 
-Let's make some typooooos.
+Let's make some typooooos. And some more tyupoes that we can fizx.
```

Comment:
```suggestion
Let's make some typos. And some more typos that we can fix.
```
I found typos

Reply:
Thanks so much, I knew that looked wrong! You are a great editor.